src/gui/mzroll/alignmentvizwidget.cpp

```diff
@@ -10,7 +10,7 @@ AlignmentVizWidget::AlignmentVizWidget(MainWindow* mw)
 void AlignmentVizWidget::plotGraph(PeakGroup*  group) {
 
     if (!_mw->alignmentVizDockWidget->isVisible()) return;
-
+    currentDisplayedGroup=group;
     intialSetup();
     PeakGroup grp = *group;
```

Comment:
@IpankajI Changing the variable names to reflect which group is the shadow group and which one is new would really help future developers

Reply:
@shubhra-agrawal changing.